### PR TITLE
Modular Abnormality Folders, and SCP-999

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds specific folders for alternative modular sets of abnos, like SCPs, Joke Abnormalities, reference abnos, etc.
Adds in a new gamemode for joke and reference abnos.
Starts cleaning up Abno sprite folders. Soon enough we will have to work on cleaning up these sprite files.

## Why It's Good For The Game
Cleanup, and support for future gamemodes and events and such
We wanted SCP-999 for the crossover event, and we got it.

Still To-do: Add in a decrepit filing cabinet for the meme stuff

## Changelog
:cl:
add: Added SCP-999
add: Added meme gamemode (Currently inaccessible
code: Started changing the organization of the EGO gear file. 
/:cl:

### CODE REVIEW

File: code/game/gamemodes/management/management.dm [hunk 1]
```diff
@@ -73,3 +73,25 @@
...
 		)
+
+
+/datum/game_mode/management/joke
+	name = "L Corp - Freaky Friday"
+	config_tag = "jokemode"
+	votable = 0
+
+	announce_span = "notice"
+	announce_text = "Manage all abnormalities not originating from Lobotomy Corporation HQ, as well as a selection of joke and reference abnormalities."
```
Comment (dsmith328) [inline code/game/gamemodes/management/management.dm:84]: This is not accurate as `ORIGIN_LOBOTOMY` is kept. Update one of the two please.

File: code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	can_breach = TRUE
+	start_qliphoth = 4
+	threat_level = ZAYIN_LEVEL
+	work_damage_amount = 4
+	work_damage_type = WHITE_DAMAGE
+
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	melee_damage_type = WHITE_DAMAGE
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm:20]: If you wanted, you could have made this a negative number and made it lose target of someone after hitting them enough times. The negative number would actually heal sanity. Not something that needs changing just a comment.

File: code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	if((heal_cooldown < world.time) && !(status_flags & GODMODE))
+		HealPulse()
+
+/mob/living/simple_animal/hostile/abnormality/tickle_monster/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, .proc/on_mob_death) // Hell
+
+/mob/living/simple_animal/hostile/abnormality/tickle_monster/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm:55]: Could probably make it have less health and add panics to this.

**Review by dsmith328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm (hunk 3 above)
Comment (Kitsunemitsu) [reply code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm:54]: Wouldn't this require a new signal?

File: code/modules/mob/living/simple_animal/abnormality/!nonstandard/joke/killsyou.dm
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/!nonstandard/joke/killsyou.dm:1]: YOOOOOOO It'S BACK

**Review by dsmith328 — CHANGES_REQUESTED** [review]
Fix merge conflicts

File: code/modules/mob/living/simple_animal/abnormality/!nonstandard/joke/killsyou.dm [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+		if(3)
+			if(user.hairstyle == "Bald")
+				death = TRUE
+
+		if(4)
+			if(user.hairstyle != "Bald")
+				death = TRUE
+
+		if(5)
+			if(user.mind.assigned_role != "Clerk")
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/!nonstandard/joke/killsyou.dm:55]: since clerks can no longer work, should be removed

**Comment by EgorDinamit** [comment]
Conflicts, if you still care for that PR, that is.

**Pull request closed without merging**